test/smoketest/project.clj

```diff
@@ -1,6 +1,6 @@
 (defproject smoketest "0.1.0-SNAPSHOT"
   :dependencies [[nrepl "1.0.0"]
-                 [cider/cider-nrepl "0.28.6"]]
+                 [cider/cider-nrepl "0.28.5"]]
```

Comment:
I had to set this back to a version that is available on Clojars to make the tests pass. Otherwise they fail because the dependency can't be downloaded. Is this supposed to test the current version?